package.json

```diff
@@ -30,7 +30,7 @@
     "eslint": "8.20.0",
     "eslint-config-next": "12.2.3",
     "postcss": "^8.4.14",
-    "tailwindcss": "^3.1.6"
+    "tailwindcss": "^3.3.6"
```

Comment:
dont need to change this but maybe ill merge a change that updates the packages so you dont have to update this every time